### PR TITLE
Add devnet connection instructions 

### PR DESCRIPTION
**Pull request by har00ga**
(Ref. #79)

This PR adds a new section on the  "Interacting with the Network" page for setting up the new live devnet, and provides some miners for users to initially choose from.

### CODE REVIEW

File: docs/how-to/build-interacting-with-the-network.md [hunk 1]
```diff
@@ -24,6 +24,17 @@ Here are some scripts to set up your own remote networks for development:
 - [Lotus Docker Image](https://github.com/openworklabs/filecoin-docker) for a simple Lotus node Docker container
 - [Filecoin Chart](https://github.com/openworklabs/filecoin-chart) for more complex Lotus node architectures managed with Kubernetes
 
+## Developer network
+In order to speed up seal times and make testing easier, a live developer network is now available for use. It features 512MB sectors, which seal much faster than in the full Testnet version.
```
Comment (mishmosh) [inline docs/how-to/build-interacting-with-the-network.md:28]:
```suggestion
In order to make testing easier, a live devnet (developer network) is now available for use. It features 512MB sectors and reduced proofs parameters, so sealing is much faster than in the full Testnet version.
```

File: docs/how-to/build-interacting-with-the-network.md [hunk 2]
```diff
@@ -24,6 +24,17 @@ Here are some scripts to set up your own remote networks for development:
 - [Lotus Docker Image](https://github.com/openworklabs/filecoin-docker) for a simple Lotus node Docker container
 - [Filecoin Chart](https://github.com/openworklabs/filecoin-chart) for more complex Lotus node architectures managed with Kubernetes
 
+## Developer network
```
Comment (mishmosh) [inline docs/how-to/build-interacting-with-the-network.md:27]:
```suggestion
## Devnet
```

File: docs/how-to/build-interacting-with-the-network.md [hunk 3]
```diff
@@ -24,6 +24,17 @@ Here are some scripts to set up your own remote networks for development:
...
+In order to speed up seal times and make testing easier, a live developer network is now available for use. It features 512MB sectors, which seal much faster than in the full Testnet version.
+
+In order to set up the developer network:
```
Comment (mishmosh) [inline docs/how-to/build-interacting-with-the-network.md:30]:
```suggestion
To connect to the Devnet:
```

**Comment by har00ga** [comment]
Did not know that! Thanks for the tip :)

**Pull request merged**